public/css/stylesheet.css

```diff
@@ -229,3 +229,72 @@ Aligns Main Menu under button
   justify-content: space-between;
 }
 /*profile page css end*/
+.form-box {
+    margin-top: 40px;
+    margin-right: 50px;
+    padding-right: 65px;
```

Comment:
get rid of `margin-top`, `margin-right`, `padding-right`. This is causing it to shrink on mobile

Reply:
so would you want me to delete .form-box completely?